src/workflow/ArcanistBulkPatchWorkflow.php

```diff
@@ -274,7 +274,11 @@ private function generateGitDiffPatches() {
       }
     }
     finally {
-      Filesystem::remove($repoPath);
+      try {
+        Filesystem::remove($repoPath);
+      } catch (Exception $ex) {
```

Comment:
Also catch Throwable, see: https://secure.phabricator.com/T12855 for details. User laptops generallu use php 7, newer OS also provide php 7 as well

Reply:
thanks, added